### DISCUSSION TITLE
Starting from the ground up

### DISCUSSION THREAD

**Discussion by PatheticMustan** [Prodigy-Hacking/ProdigyMathGameHacking#676] [(D) Developer]
We made our hacks through a series of slow increments, and bugfixes. There's a lot of remaining bloat, and questionable architecture changes we've made. Starting from the ground up will take a lot of work, but will also give us more freedom.

Let's learn from our mistakes, and keep the good things we already have.

we don't literally have to start from scratch, but rewrites of certain things may be good

Basic requirements for hacks:
- Modifying game.min and public game.min with an array of updates
- Something to inject these modified files
- A graphical cheat menu

Current issues:
- WCM takes a long time to build
- Too many things
- Iffy support on school computers, or devices with a filter.
- Update process is trash
- Speed of updates are trash
- No index page on PHEx

**Comment by NessiePUYOY** [comment]
Its ok, your doing a great job Mustan! Look at how much work you have done and you never gave up did you? I see not just believe in yourself and take time you can do it :)

**Reply by ghost** [reply]
we still need to clean it up a bit

**Reply by hellonearth311** [reply]
The comments in the syntax clog WCM so it's impossible to see what's going on.
I'll look into deleting these.

**Reply by PatheticMustan** [reply]
no?????????????????????????????????????
